### ISSUE TITLE
libGL Error Steam crash

### ISSUE THREAD

**Issue by feldim2425** [ValveSoftware/steam-for-linux#7499]
#### Your system information

* Steam client version (build number or date): 1.0.0.67 (downloaded it on 1.Dez)
* Distribution: Ubuntu 20.04.1 LTS
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes (everything up to data)
* Nvidia drivers: 455.45.01 (according to nvidia-smi; nvidia-driver-455 package)

#### Please describe your issue in as much detail as possible:

Steam crashes. The updater still runs but since last update it fails to open the main window.
I tried updating the driver from 450 to 455 and running steam with ``stream --reset`` but without luck.
Both glxgears and glxdemo run without issues. 
Since I recently switched to a two monitor setup I also tried to unplug one of them to check if that was the issue but Steam still crashes.

I added the ``LIBGL_DEBUG=verbose`` environment variable to get some more info:
[Console Output](https://gist.github.com/feldim2425/08f0081036ac13db175fd99d7c026084)

#### Steps for reproducing this issue:

1. Download & Install Steam
2. Run Steam

**Comment by kisak-valve** [comment]
Hello @feldim2425, swrast is mesa's slower cpu opengl renderer and the fact Steam is trying to use it and failing hints that you're using the nVidia proprietary driver, but don't have nVidia's 32 bit userspace installed.

Can you check the 32 bit userspace is installed with something like `apt policy libnvidia-gl-455:i386`? If it's not installed, you can run something like `sudo apt install libnvidia-gl-455:i386` to add it to your system.

**Comment by feldim2425** [comment]
Problem is if I try to install ``libnvidia-gl-455:i386`` apt will remove the actual driver. Installing both the 64-bit version and the 32-bit version doesn't seem to work.
```
$ sudo apt install libnvidia-gl-455:i386 nvidia-driver-455
Reading package lists... Done
Building dependency tree       
Reading state information... Done
nvidia-driver-455 is already the newest version (455.45.01-0ubuntu1).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libnvidia-gl-455 : Breaks: libnvidia-gl-455:i386 (!= 455.45.01-0ubuntu1) but 455.45.01-0ubuntu0~0.20.04.1 is to be installed
 libnvidia-gl-455:i386 : Breaks: libnvidia-gl-455 (!= 455.45.01-0ubuntu0~0.20.04.1) but 455.45.01-0ubuntu1 is to be installed
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
```

**Comment by kisak-valve** [comment]
Debian's multiarch implemention requires that the 32 bit and 64 variants of a package have exactly the same version. Ubuntu inherits this behavior. You can resolve this by checking what is available in the repos with something like `apt policy policy libnvidia-gl-455 libnvidia-gl-455:i386` and explicitly telling apt to install the common version between the two, but it's fishy that "libnvidia-gl-455 455.45.01-0ubuntu1" is around in the first place. A [quick search](https://launchpad.net/ubuntu/+source/nvidia-graphics-drivers-455/455.45.01-0ubuntu1) indicates this version is built for the not-yet-released Ubuntu 21.04 release and that would not be intended to be used with an LTS release.

**Comment by feldim2425** [comment]
Oh thanks.
It worked but I had to add expand this to some more libraries since it refused to install and/or downgrade.
Basically ended up with this command, removing all the packages would probably have been easier but what's done is done. Thanks
```
sudo apt install libnvidia-gl-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-gl-455:i386=455.45.01-0ubuntu0~0.20.04.1 nvidia-driver-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-cfg1-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-fbc1-455=455.45.01-0ubuntu0~0.20.04.1 xserver-xorg-video-nvidia-455=455.45.01-0ubuntu0~0.20.04.1 nvidia-utils-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-encode-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-decode-455=455.45.01-0ubuntu0~0.20.04.1 nvidia-compute-utils-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-extra-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-compute-455=455.45.01-0ubuntu0~0.20.04.1 nvidia-kernel-source-455=455.45.01-0ubuntu0~0.20.04.1 libnvidia-ifr1-455=455.45.01-0ubuntu0~0.20.04.1
```
